src/sections/ThemesSection/ThemesSection.jsx

```diff
@@ -1,25 +1,25 @@
-import React, { useEffect } from "react";
-import "./ThemesSection.css";
-import redbtn from "../../assets/themes/redbtn.png";
-import themedata from "./ThemeData";
-import doge from "../../assets/img/doge.png";
+import React from 'react';
+import './ThemesSection.css';
+import { motion } from 'framer-motion';
 
 const ThemesSection = () => {
   return (
-    <>
-      <div className="themesection_god">
-        <div className="themesection_parent" id="themes">
-          <h1 className="themesection_title">Themes</h1>
-          <hr style={{ marginBottom: "70px" }} />
-
-          <h3
-            style={{ textAlign: "center", color: "white", fontWeight: "700" }}
-          >
-            To be decided soon...
-          </h3>
+    <motion.div
+      initial={{ opacity: 0 }}
+      whileInView={{ opacity: 1, transition: { duration: 1 } }}
+      viewport={{ once: false }}
+      className="hack4bengal__section5"
+    >
+      <div className="h4b_about_main">
```

Comment:
As of the issue mentioned i don't think we might be needing framer motion's tool to get it done, i guess we can revert this to have a normal div in place.